src/pretix/control/templates/pretixcontrol/checkin/simulator.html

```diff
@@ -0,0 +1,130 @@
+{% extends "pretixcontrol/items/base.html" %}
+{% load i18n %}
+{% load bootstrap3 %}
+{% load escapejson %}
+{% load getitem %}
+{% load static %}
+{% load compress %}
+{% block title %}{% trans "Check-in simulator" %}{% endblock %}
+{% block inside %}
+    <h1>{% trans "Check-in simulator" %}</h1>
```

Comment:
I feel like the simulator is a little bit "disconnected" from the selected checkin-list.

Perhaps we can try to have the headline the same or similar to the checkinlist positions- oder edit-view; at least with the name of the list, possibly even with buttons to edit the list/return to its main view.